compiler/ext/config.ml

```diff
@@ -1,9 +1,23 @@
 let version = "4.06.1+BS"
 
+(* FIXME: Unreliable resolution *)
```

Comment:
Still the case?

Reply:
I tried here, but completely removing the `standard_library` entry point would be trickier than I thought. Maybe it's better to continue it from another PR.